README.md

```diff
@@ -1,59 +1,170 @@
+[![Licensed under the Apache 2.0 open-source license](https://img.shields.io/badge/License-Apache%202.0-3c60b1.svg?logo=opensourceinitiative\&logoColor=white\&style=flat-square)](https://github.com/quantumlib/qualtran/blob/main/LICENSE)
+[![Compatible with Python versions 3.10 and higher](https://img.shields.io/badge/Python-3.10+-6828b2.svg?style=flat-square&logo=python&logoColor=white)](https://www.python.org/downloads/)
+[![Qualtran project on PyPI](https://img.shields.io/pypi/v/qualtran.svg?logo=python&logoColor=white&label=PyPI&style=flat-square&color=9d3bb8)](https://pypi.org/project/qualtran)
 
-## Documentation
+[Installation](#installation) &ndash;
+[Usage](#usage) &ndash;
+[Documentation](#documentation) &ndash;
+[Community](#community) &ndash;
+[Citation](#citation) &ndash;
+[Contact](#contact)
 
-Documentation is available at https://qualtran.readthedocs.io/
+</div>
+
+Qualtran (_quantum algorithms translator_) is a set of abstractions for representing quantum
+programs and a library of quantum algorithms expressed in that language to support quantum
+algorithms research.
 
 ## Installation
 
-Qualtran is being actively developed. We recommend installing from source:
+Qualtran is being actively developed. We recommend installing from the source code.
 
-For a local editable copy:
+The following commands will clone a copy of the repository, then install the Qualtran package in
+your local Python environment as a local editable copy:
 
-    git clone https://github.com/quantumlib/Qualtran.git
-    cd Qualtran/
-    pip install -e .
+```shell
+git clone https://github.com/quantumlib/Qualtran.git
+cd Qualtran/
+pip install -e .
+```
 
 You can also install the latest tagged release using `pip`:
 
-    pip install qualtran
+```shell
+pip install qualtran
+```
+
+You can also install the latest version of the main branch on GitHub:
+
+```shell
+pip install git+https://github.com/quantumlib/Qualtran
+```
+
+## Usage
+
+> [!WARNING]
+> Qualtran is an experimental preview release. We provide no backwards compatibility guarantees.
+> Some algorithms or library functionality may be incomplete or contain inaccuracies. Open issues or
+> contact the authors with bug reports or feedback.
+
+### Python interpreter and programs
+
+You should be able to import the `qualtran` package into your interactive Python environment as
+as well as your programs:
+
+```shell
+import qualtran
+```
+
+If this is successful, you can move on to learning how to
+[write bloqs](https://qualtran.readthedocs.io/en/latest/_infra/Bloqs-Tutorial.html) or investigate
+the [bloqs library](https://qualtran.readthedocs.io/en/latest/bloqs/index.html#bloqs-library).
+
+### Physical Resource Estimation GUI
+
+Qualtran provides a GUI for estimating the physical resources (qubits, magic states, runtime, etc.)
+needed to run a quantum algorithm. The GUI can be run locally by running:
+
+```shell
+cd $QUALTRAN_HOME
+python -m qualtran.surface_code.ui
+```
```

Comment:
we should probably remove this from the readme to avoid confusion with the forthcoming web app